CMakePresets.json.license

```diff
@@ -0,0 +1,4 @@
+SPDX-FileCopyrightText: 2021, 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
```

Comment:
I find the year here curious, should it not just be 2023?

Reply:
Also squash the second commit into the first so each commit passes reuse.

Reply:
I just copied the file from the repo somewhere else.
Is the year even necessary? I saw some oss project that stopped writing the year.

Reply:
Right, https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects and plenty of other large tech vendors have started dropping the year.

For now in this project we have had at least the year the file was introduced, lets be consistent for now and keep adding the year the file was made. Maybe later we can decide to purge the year from everywhere in it's own change :-)

Reply:
(and more articles linked here https://hynek.me/til/copyright-years/ )

Reply:
Oh yes that's the one I've stump on across